tests/improver-nbhood/00-null.bats

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bats
+
+@test "nbhood no arguments" {
```

Comment:
That is a good check! Yes, the plan is to have site-specific reference input and output files and use those as CLI acceptance tests. They are unlikely to span the full range of functionality of most of our plugins, but a good combination of CLI and unit tests should do that.

I can ad a `-h` one.